content/docs/learning-guide/tutorials/remote-repositories/create-remote-repository.md

```diff
@@ -0,0 +1,193 @@
+
+In O3DE we provide the option for users to create a release archive with .zip format.<br>
+
+Due to user safty, we use `sha-256` to hash our files before creating the release archive. Thus, you won't be able to modify your `repo.json` file once your archive is created. Every time you make a change to your gem, including the repo.json file, the `sha-256` hash will change.<br>
+
+<ul>
+
+Note: You must know where your zip will be created online **download path** before you actually create the release archive zip.</ul>
+<br>
+<ul>
+
+### Add A Gem To Your Remote Repository And Create A Release Archive
+---
+Here is a Github example of how to create a release archive. <br><br>
+Note: `--release-archive-path` command can only be used after you call the `--add-gem` command.<br> See below example:
+
+<ul>
+
+<li>
+
+To create a release archive `download prefix`, make sure you follow this pattern for any Github repositories:<br>
+<ul>
+
+ `https://github.com/YourGitAccount/RemoteProjectName/releases/download/YouReleaseTag`<br>
+
+```
+o3de.bat edit-repo-properties --repo-path "C:\o3de_Projects\<RemoteProjectName>\repo.json" --add-gem "C:\o3de_Projects\<RemoteProjectName>\Gems\MyGem" --release-archive-path "C:\o3de_Projects\<RemoteProjectName>\Gems" --download-prefix "https://github.com/<YourGitAccount>/<RemoteProjectName>/releases/download/<YouReleaseTag>"
+```
+<ul>
+
+ Provide the following required information:
+ <li> --repo-path: the absolute path to your remote project's repo.json file
+ <li> --add-gem: the path to the directory to where the actual gem you want to add is located
+ <li> --release-archive-path: the path to where you want to place your .zip archive, it should be placed in the Gems folder of your remote project folder
+ <li> --download-prefix: the URI where your gem archive can be download from
+
+<br>
+1. Here is an example of where you should upload your release to Github (you can find this on your version control url):
+
+
+![tag](Media/add_release.png)
+
+2. Here is where you would be setting your release tag (make sure this tag is the same as the last parameter you passed for the `--download-prefix` arg)
+
+![tag](Media/release_tag.png)
+
+ </ul>
+</ul>
+
+WARNING: Make sure you know where your release will be available after uploading it to GitHub. Once uploaded, you won't be able to change the `download_source_uri` in your `repo.json` file. If you input the wrong download path, you will need to repeat this step and rezip the file and upload the release to Github again.
+</ul>
+After you upload your gem to the version control website, other users will be able to download it in their O3DE engine. It will be available in the Project Manager. Users can navigate to Remote Sources and use your remote project url to download associated gems based on the download path you provided in this step.<br><br>
```

Comment:
```suggestion
After you upload your Gem to the version control website, other users will be able to download it in their O3DE engine. It will be available in the Project Manager. Users can navigate to Remote Sources and use your remote project URL to download associated Gems based on the download path you provided in this step.
```